src/components/CouponDetails/index.jsx

```diff
@@ -56,6 +56,7 @@ class CouponDetails extends React.Component {
       isCodeAssignmentSuccessful: undefined,
       isCodeReminderSuccessful: undefined,
       isCodeRevokeSuccessful: undefined,
+      doesCodeActionHaveErrors: undefined,
```

Comment:
`hasCodeActionErrors` sounds more better?

Reply:
I think `doesCodeActionHaveErrors` is better as it conforms to the style of boolean variable names defined above.